src/driver/amdxdna/amdxdna_mailbox.c

```diff
@@ -590,7 +591,7 @@ xdna_mailbox_create_channel(struct mailbox *mb,
 			    u32 iohub_int_addr,
 			    int mb_irq)
 {
-	struct mailbox_channel *mb_chann;
+	struct mailbox_channel *mb_chann = {};
```

Comment:
Why not mb_chann = NULL?

Reply:
same as previous, it's usually more condensed which is why I typically use it.  And if you are automating a number of fixes it can save you the hassle of resolving the type info.

Reply:
Same as before, could you follow the existing coding style?